addons/todo/static/src/views/todo_form/todo_form_controller.js

```diff
@@ -1,14 +1,55 @@
 /** @odoo-module **/
 
+import { _lt } from "@web/core/l10n/translation";
 import { FormController } from "@web/views/form/form_controller";
 import { TodoEditableBreadcrumbName } from "@todo/components/todo_editable_breadcrumb_name/todo_editable_breadcrumb_name";
+const { onWillStart } = owl;
 
 /**
  *  The FormController is overridden to be able to manage the edition of the name of a to-do directly
  *  in the breadcrumb.
  */
 
-export class TodoFormController extends FormController {}
+export class TodoFormController extends FormController {
+    setup() {
+        super.setup();
+        onWillStart(async () => {
+            this.conversionViewId = await this.model.orm.call(
+                'project.task',
+                'get_conversion_view_id',
+                [],
+            );
```

Comment:
Cannot we do that rpc call in `callback` function of `convert to task` button instead of doing a rpc call each time we do to the todo form view. 🤔 

If the callback function can be async then we could do that call there and so we will just do an additional rpc call when the user clicks on `Convert to Task` button.